hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/ECReplicationConfig.java

```diff
@@ -82,14 +82,17 @@ public ECReplicationConfig(int data, int parity, EcCodec codec,
   }
 
   /**
-   * Create an ECReplicationConfig object from a string representing the
-   * various parameters. Acceptable patterns are like:
+   * Create an {@link ECReplicationConfig} object from a string representing
+   * the various parameters. Acceptable patterns are like:
    *     rs-3-2-1024k
    *     RS-3-2-2048k
    *     XOR-10-4-4096K
    * IllegalArgumentException will be thrown if the passed string does not
    * match the defined pattern.
-   * @param string
+   * @param string This is a string parameter used to create an
+   *        {@link ECReplicationConfig} instance. It must meet the
+   *        requirements, otherwise {@link IllegalArgumentException} will be
+   *        thrown.
```

Comment:
I think the same is already described in the existing javadoc.  No need to repeat it in different words.

Reply:
Thanks @adoroszlai  for the comment and review.
I will update later.